content/handbook/support/_index.md

```
@@ -0,0 +1,631 @@
+- 興味を持つ他のチームメンバーと外部プロジェクトやタスクで協働する機会を持つ
+- メンバー間で重複作業が発生するのを避ける
+
+GitLab、私たちの FOSS プロジェクト、または GitLab コンポーネントのドキュメントや機能に関する Issue は、この Issue トラッカーには入れず、適切な Issue トラッカーに登録すべきです。
+
+実行可能な解決策の提案がある場合、[マージリクエストから始める](/handbook/communication/#start-with-a-merge-request) のがベストです。チームをタグ付けしてフィードバックを求め、[Support Week in Review](#support-week-in-review) でリンクしてください。
+
+### Slack
+
+私たちはチームコミュニケーションのために、GitLab の [Slack 利用に関する一般ガイドライン](/handbook/communication/#slack) に従います。アクティビティは 90 日間しか保持されないため、重要な情報はチームのハンドブック、製品ドキュメント、Issue トラッカー、お客様のチケットに必ず移動してください。
+
+#### spt_ と support_ プレフィックスの使い分け
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)